src/Evolution/Systems/Cce/WorldtubeDataManager.hpp

```diff
@@ -48,6 +48,7 @@ class BondiWorldtubeDataManager;
  *   underlying data source. This is primarily used for monitoring the frequency
  *   and size of the buffer updates.
  */
+template <typename BoundaryTags>
 class WorldtubeDataManager : public PUP::able {
```

Comment:
Add a line in the dox that `BoundaryTags` is a `tmpl::list` of tags on the worldtube boundary